Explorer/Assets/DCL/Multiplayer/Profiles/RemoveIntentions/RemoveIntention.cs

```diff
@@ -16,5 +17,14 @@ public RemoveIntention(string walletId, RoomSource fromRoom)
 
         public override string ToString() =>
             $"(RemoveIntention: (WalletId: {WalletId}, From Room: {FromRoom}))";
+
+        public bool Equals(RemoveIntention other) =>
+            WalletId == other.WalletId && FromRoom == other.FromRoom;
```

Comment:
Should we consider the room? Or only the wallet?

Reply:
Where is it used?

Reply:
Why did it work before?